src/main/java/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -41,6 +41,15 @@ protected ResponseEntity<Object> handleEntityNotFoundException(
                 .body(ex.getMessage());
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    protected ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex
+    ) {
```

Comment:
```suggestion
    protected ResponseEntity<Object> handleRegistrationException(RegistrationException ex) {
```

Reply:
It`s magic!) I thought it was a very big row